CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 Change log
 -----------
 
+* (backport) kernel-modules-headers: specify cross compiler command [Florin]
```

Comment:
Any advantage in having this tag here? Basically I would expect most of the things to be backported from 2.x as  teh main dev is done there now.